AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_CubeMapCaptureAdded.py

```diff
@@ -108,39 +100,45 @@ def AtomEditorComponents_CubeMapCapture_AddedToEntity():
 
-        # 5. Set Specular IBL property: Very High
-        cubemap_capture_component.set_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL'), value=SPECULAR_IBL_QUALITY['Very High'])
-        Report.result(Tests.specular_ibl_property_set_to_very_high, cubemap_capture_component.get_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL')) == SPECULAR_IBL_QUALITY['Very High'])
-
-        # 6. Set Specular IBL property: Very Low
-        cubemap_capture_component.set_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL'), value=SPECULAR_IBL_QUALITY['Very Low'])
-        Report.result(Tests.specular_ibl_property_set_to_very_low, cubemap_capture_component.get_component_property_value(AtomComponentProperties.cube_map_capture('Specular IBL')) == SPECULAR_IBL_QUALITY['Very Low'])
-
-        # 7. Set Exposure property
+        # 5. Test Specular IBL CubeMap Quality values
+        for quality in SPECULAR_IBL_QUALITY:
+            test_quality = (
+                f"Specular IBL quality set too {quality}",
+                f"P1: failed to set 'Specular IBL quality' to {quality}")
+            cubemap_capture_component.set_component_property_value(
+                AtomComponentProperties.cube_map_capture('Specular IBL CubeMap Quality'), SPECULAR_IBL_QUALITY[quality])
+            Report.result(test_quality, cubemap_capture_component.get_component_property_value(
+                AtomComponentProperties.cube_map_capture('Specular IBL CubeMap Quality')) == SPECULAR_IBL_QUALITY[quality])
+
+        # 6. Set Exposure property
         cubemap_capture_component.set_component_property_value(AtomComponentProperties.cube_map_capture('Exposure'), 16.0)
         get_exposure_property = cubemap_capture_component.get_component_property_value(AtomComponentProperties.cube_map_capture('Exposure'))
         Report.result(Tests.exposure_property_set, get_exposure_property == 16.0)
 
-        # 8. Set Capture Type property: Diffuse IBL
-        cubemap_capture_component.set_component_property_value(AtomComponentProperties.cube_map_capture('Capture Type'), value=CUBEMAP_CAPTURE_TYPE['Diffuse ILB'])
-        Report.result(Tests.capture_type_property_set_to_Diffuse_ibl, cubemap_capture_component.get_component_property_value(AtomComponentProperties.cube_map_capture('Capture Type')) == CUBEMAP_CAPTURE_TYPE['Diffuse ILB'])
+        # 7. Set Capture Type property: Diffuse IBL
+        cubemap_capture_component.set_component_property_value(AtomComponentProperties.cube_map_capture(
+            'Capture Type'), value=CUBEMAP_CAPTURE_TYPE['Diffuse ILB'])
+        Report.result(
+            Tests.capture_type_property_set_to_Diffuse_ibl,
+            cubemap_capture_component.get_component_property_value(AtomComponentProperties.cube_map_capture(
+                'Capture Type')) == CUBEMAP_CAPTURE_TYPE['Diffuse ILB'])
```

Comment:
it's harder to fit the newlines in this for readability, but I prefer to have AtomComponentProperties.cube_map_capture('Capture Type') all on one line.
avoid starting line continuation with unexplained strings or ==
you could copy/paste the line in and see how pycharm breaks it up for line length automatically

Reply:
Updated